### PR TITLE
Release notes and doc updates for 2022.03.0

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Add release notes and doc updates for 2022.03.0
- **Is this PR related to a Monolith release?** Yes, 2022.03.0

### Resolves:

* [OKTA-471353](https://oktainc.atlassian.net/browse/OKTA-471353)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md [hunk 1]
```diff
@@ -46,88 +53,191 @@ The [Brands API](/docs/reference/api/brands/) is a feature (currently in Early A
...
+- some of the org attributes in these variables
 
-The templating syntax that is used in enhanced email macros is the Velocity Templating Language (VTL). Variables that are to be interpolated in the content of the template are preceded by a dollar sign. Dot notation is used to reference sub-objects.
+Email templates use common and unique VTL variables. When you interpolate variables in the template content, precede them with a dollar sign. Use dot notation to reference sub-objects.
 
-The new templating syntax is different from the EL expression-based Okta email templating syntax that was previously used. You no longer need to use curly braces around the variable name.
+For example, reference the first name of a user with `${user.profile.firstName}`.
 
-Previously with EL syntax, you could reference the first name of the user by using `${user.firstName}`. Now with the Velocity Templating syntax, you would use `$user.profile.firstName`.
+See [Velocity Templating Language](https://help.okta.com/okta_help.htm?type=oie&id=ext-velocity-variables) for available template variables.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md:141]:
```suggestion
See [Use VTL variables](#use-vtl-variables) for available email template variables.
```

File: packages/@okta/vuepress-site/docs/guides/customize-themes/index.md [hunk 2]
```diff
@@ -10,12 +10,16 @@ This guide explains how to use the Brands API to rapidly customize the theme of
 
 **Learning outcomes**
 
-Use the Brands API to customize the theme of your Okta org.
+- Use the Brands API to customize the theme of your Okta org.
+- Customize the email templates of your Okta org.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-themes/index.md:14]:
```suggestion
- Use the Brands API to customize the email templates of your Okta org.
```

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 3]
```diff
@@ -3,6 +3,47 @@ title: Okta Identity Engine API Products release notes 2022
...
+
+The user-authenticated epoch timestamp is provided as the `auth_time` [claim in the access token](/docs/reference/api/oidc/#reserved-claims-in-the-payload-section).
+
+#### Custom Administrator Roles is GA in Production
+
+The Okta [Custom Administrator Roles](/docs/reference/api/roles/) API provides operations that you can use to create customized roles and assign them to a user or a group.
+
+#### Email Address Bounces API is EA in Preview
+
+Okta admins can now control the bounced email address list through the Email Address Bounces API. When Okta-sent email addresses are blocked from an email service (the bounced email list), admins can use this API to create a list of blocked email addresses to be removed from the email service.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:29]: Should we link to the API docs, to help the devs reading these notes? or should we not if it's only EA in preview?

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 4]
```diff
@@ -1,6 +1,46 @@
...
+
+The user-authenticated epoch timestamp is provided as the `auth_time` [claim in the access token](/docs/reference/api/oidc/#reserved-claims-in-the-payload-section).
+
+#### Custom Administrator Roles is GA in Production
+
+The Okta [Custom Administrator Roles](/docs/reference/api/roles/) API provides operations that you can use to create customized roles and assign them to a user or a group.
+
+#### Email Address Bounces API is EA in Preview
+
+Okta admins can now control the bounced email address list through the Email Address Bounces API. When Okta-sent email addresses are blocked from an email service (the bounced email list), admins can use this API to create a list of blocked email addresses to be removed from the email service.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022/index.md:28]: See comment in OIE RNs.

**Review by brentschaus-okta — APPROVED** [review]
LGTM!

A big one :)

Just one small question in the RNs.

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md (hunk 3 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:29]: The API docs are missing for this new endpoint. Janet is going to follow up with the Dev Mgr to ensure that we have API docs for this.

**Pull request merged**